libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -244,4 +255,8 @@ export class SkyAgGridWrapperComponent
       }
     }
   }
+
+  #getHeaderHeight(): number {
```

Comment:
Instead of duplicating the header height logic in the component, maybe we can call into the service somehow?

Reply:
Good call. Updated.